vtex/faststore
# Feat: Adds `QuantitySelector` component

## What's the purpose of this pull request?

- [x] Moves QuantitySelector component to @faststore/components
- [x] Moves QuantitySelector styles on @faststore/ui
- [x] Adds basic QuantitySelector styles on @faststore/ui
- [x] Adds QuantitySelector doc page

How to test it?

run `yarn workspace site dev` and check the QuantitySelector component  doc page or test it using the doc preview link: [QuantitySelector Component](https://faststore-site-git-feat-quantity-selector-comp-77a068-faststore.vercel.app/components/molecules/quantity-selector)

### Starters Deploy Preview

[Feat: Replaces QuantitySelector FSUI #344](https://github.com/vtex-sites/nextjs.store/pull/344)

<img width="32" alt="slider_controler_left" src="https://user-images.githubusercontent.com/3356699/211866912-a53e5a66-506e-4752-bbc3-b50089fd8fce.png">


❗ The current input behavior is not the ideal! It will be addressed in another task.
